src/main/frontend/pipeline-console-view/pipeline-console/main/PipelineConsole.tsx

```diff
@@ -160,12 +163,8 @@ export class PipelineConsole extends React.Component<
 
   // State update methods
   async updateState() {
-    let stagesPromse = this.setStages();
-    let stepsPromise = this.setSteps();
-    // Try waiting for
-    await stagesPromse;
-    await stepsPromise;
-    this.handleUrlParams();
```

Comment:
This didn't always work. `setState` is asynchronous and often the steps weren't yet set when running `handleUrlParams()`.